docs/contributing/building.rst

```diff
@@ -135,6 +135,63 @@ or source it into your shell,
 
    . $BUILD_DIR/python_env.sh
 
+Resolving Common Build Issues
```

Comment:
Also note that if you try anaconda vs miniconda3 it will not work. I learned this the hard way. Maybe worth mentioning this at the end?